vue-frontend/src/components/partials/NewHeader.vue

```diff
@@ -33,50 +33,77 @@
-                      ? navbar.showContactBtn
-                        ? ''
-                        : 'after:tw-w-full after:tw-scale-x-0 after:tw-scale-x-100'
-                      : '',
+                        ' after:tw-absolute after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-w-full after:tw-h-[3px]  after:tw-bg-gradient-[90deg] after:tw-from-[#f2709c] after:tw-to-[#ff9472] after:tw-origin-bottom-left after:tw-duration-300 after:tw-scale-x-0 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left hover:tw-bg-clip-text hover:tw-bg-gradient-[270.11deg] hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-text-transparent ',
                   ]"
                 >
                   <span
                     :class="[
-                      currentRoutePath == navbar.url && !navbar.showContactBtn
-                        ? 'v2-canopas-gradient-text'
-                        : '',
+                      navbar.id == 7 ? 'hover:v2-canopas-gradient-text' : '',
                     ]"
-                    class="tw-flex tw-mt-[0.1rem]"
                     >{{ navbar.name }}</span
-                  ></router-link
-                >
+                  >
+                </router-link>
                 <a
                   v-else
                   class="tw-inline-block tw-relative tw-mr-[20px] sm:tw-mr-[30px] lg:tw-mr-[8px] xl:tw-mr-[30px] after:tw-absolute after:tw-w-full after:tw-h-[3px] after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-bg-gradient-[90deg] after:tw-from-[#f2709c] after:tw-to-[#ff9472] after:tw-origin-bottom-left after:tw-duration-300 after:tw-scale-x-0 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left hover:tw-bg-clip-text hover:tw-bg-gradient-[270.11deg] hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-text-transparent"
                   :href="navbar.url"
                   :target="navbar.target"
                   >{{ navbar.name }}</a
                 >
+                <ul
+                  v-show="showContributionMenu"
+                  @mouseleave="showContributionMenu = false"
+                  class="tw-absolute xl:tw-left-[296px] 2xl:tw-left-[475px] lg:tw-top-[4.5rem] tw-w-max tw-border tw-rounded-[5px] tw-bg-white tw-shadow-md tw-px-6 tw-py-[1.5rem] tw-flex-col tw-space-y-6"
+                  :class="
+                    subMenus.length == 2
+                      ? 'lg:tw-left-[180px]'
+                      : 'lg:tw-left-[169px]'
+                  "
+                >
+                  <li
+                    v-for="navbar in subMenus"
+                    :key="navbar"
+                    class="tw-relative tw-group"
+                  >
+                    <a
+                      class="tw-group tw-relative tw-py-[1rem] tw-font-inter-medium tw-text-black-core/[0.6] tw-text-[1rem] md:tw-text-[1.09375rem] lg:tw-text-[1.188rem] tw-leading-[1.125rem] md:tw-leading-[1.28125rem] lg:tw-leading-[1.437rem] group-hover:tw-text-white group-hover:tw-z-[1]"
```

Comment:
It's not vertically aligned when there is a gradient background. Use flex to make it aligned